src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -94,10 +97,7 @@ describe('form:', () => {
       mappedIdsSet.size,
       'a schema may have been removed from vets-json-schema/dist/schemas',
     );
-    expect(includedSchemaIdsSet.size).to.not.greaterThan(
-      mappedIdsSet.size,
-      'a schema may have been added to vets-json-schema/dist/schemas',
-    );
```

Comment:
Did you mean to remove this test?

Reply:
No not at all don't know how this happened will revert!